docs/docs/building-from-source.md

```diff
@@ -5,15 +5,12 @@ Users are recommended to use the [pre-built xemu binaries](https://github.com/mb
 ## Windows
 
 Windows builds are cross-compiled from Ubuntu. If you would like to build *on* Windows, you can use WSL2 and Docker. See [official Docker
-documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up.
+documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up. The instructions are below for windows. Make sure to be using the wsl2 terminal for the instructions.
```

Comment:
Don't need the line `The instructions are below for windows.`

Reply:
Resolved on latest commit, Is that better?